common/data_refinery_common/utils.py

```diff
@@ -296,14 +337,16 @@ def calculate_sha1(absolute_file_path):
 
     return hash_object.hexdigest()
 
+
 def get_sra_download_url(run_accession, protocol="fasp"):
     """Try getting the sra-download URL from CGI endpoint"""
-    #Ex: curl --data "acc=SRR6718414&accept-proto=fasp&version=2.0" https://www.ncbi.nlm.nih.gov/Traces/names/names.cgi
+    # Ex: curl --data "acc=SRR6718414&accept-proto=fasp&version=2.0" \
+    #   https://www.ncbi.nlm.nih.gov/Traces/names/names.cgi
     cgi_url = "https://www.ncbi.nlm.nih.gov/Traces/names/names.cgi"
     data = "acc=" + run_accession + "&accept-proto=" + protocol + "&version=2.0"
     try:
         resp = requests.post(cgi_url, data=data)
-    except Exception as e:
```

Comment:
I'm guessing you changed this because `e` was unused.

Reply:
yup